### ISSUE TITLE
Am I tied to Node when using Mimosa?

### ISSUE THREAD

**Issue by jeffijoe** [dbashford/mimosa#283]
So I got Mimosa installed yesterday and tried to set up a new project. I will be using PHP in my views, as I am tied to a shared Linux webhost.

Running Apache (Xampp) on my Windows machine, I'd like to develop that project blazingly fast using all the awesome Mimosa provides. Thing is, once Mimosa spit out the views and assets folder, I don't know where to start, it seems Mimosa is hinting towards using pure JS. Should I just delete the views folder at create an index.php in my project root? Do I need to have a flag in my PHP code that determines if I am in dev or production mode, so I know when to use the built assets or the dev ones?

Long story short: For a plain-vanilla PHP "site", where do I start?
